src/tap.h

```diff
@@ -92,9 +94,14 @@ class TestResult {
   }
 
   std::string toString() const {
-    std::stringstream ss;
-    ss << this->status << " " << this->number << " " << this->name << " "
-       << this->getComment();
+    ss << this->status << " " << this->number << " " << this->name;
+    std::string comment_text = this->getComment();
+    if (!comment_text.empty()) {
```

Comment:
That will change the current behaviour of the library. If you have time, take at look at the GTest API and check how to add a configuration option to enable this diagnostic output or keep the default behaviour.